2_Operations/user-stories.md

```diff
@@ -1,29 +1,23 @@
-# Guide for user stories
+# User Stories
 
-## Writing a user story
+User Stories are well-defined tasks to be done by a Nitean that have a specific, measurable goal and impact. The large majority of issues are written as User Stories and only issues created by support that are reactionary and straightforward are not. Support issues are also not included in sprints.
+
+## Writing a User Story
+
+To create a new user story, open a [new issue](https://github.com/niteoweb/operations/issues/new) in the Operations repository and you will find a template is provided to guide you through the process. If you need to create a user story in another repository you will need to copy and paste over this Operations template.
 
-To create a new user story, open a [new issue][Operations new issue] in the Operations repository and you will find a template is provided to guide you through the process. If you need to create a user story in another repository you will need to copy and paste over this Operations template.
 
 ### Summary
 
 The opening lines of a user story are to be written in simplified, layman's terms for the whole team to understand. The viewpoint of a persona is to be used to fully understand the use-case.
 
  > As a *persona*, I want ..., so that ...
 
-Some frequently used personas:
-
- * Nitean
- * Nitean on support
- * Developer
- * Marketer
- * DevOp
-
-An end-user story should be written using the product's personas.
+Whenever possible, think about the end user (product persona) and how the result impacts them. Other possible personas are Nitean, Support, Developer, Marketer and DevOps. That said, very rarely is something not done directly for the user.
```

Comment:
Can we have a link to those personas defined in intra

Reply:
Yes. WooCart has them in project repo, EBN will have too soon.